src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "altinn-app-frontend",
-  "version": "3.17.1",
+  "version": "3.17.2",
```

@ivarne final adjustment 

```suggestion
  "version": "3.18.0",
```

---

@ivarne never mind, I could fix that my self...